src/user/screens/repository-list.screen.js

```diff
@@ -131,22 +194,67 @@ class RepositoryList extends Component {
       <ViewContainer>
         <View>
           <Header>
-            <SearchBarWrapper>
-              <SearchContainer>
-                <SearchBar
-                  textColor={colors.primaryDark}
-                  textFieldBackgroundColor={colors.greyLight}
-                  showsCancelButton={searchFocus}
-                  onFocus={() => this.setState({ searchFocus: true })}
-                  onCancelButtonPress={() =>
-                    this.setState({ searchStart: false, query: '' })}
-                  onSearchButtonPress={query => {
-                    this.search(query);
-                  }}
-                  hideBackground
-                />
-              </SearchContainer>
-            </SearchBarWrapper>
+            {this.state.searchFocus ? (
+              <SearchBarWrapper>
+                <SearchContainer>
+                  <SearchBar
+                    textColor={colors.primaryDark}
+                    textFieldBackgroundColor={colors.greyLight}
+                    showsCancelButton={searchFocus}
+                    onCancelButtonPress={() =>
+                      this.setState({
+                        searchStart: false,
+                        searchFocus: false,
+                        query: '',
+                      })
+                    }
+                    onSearchButtonPress={query => {
+                      this.search(query);
+                    }}
+                    hideBackground
+                    autoFocus
+                  />
+                </SearchContainer>
+              </SearchBarWrapper>
+            ) : (
+              <StyledButtonGroup
```

Comment:
English labels fits right in, but I'm afraid other translations won't play as nice..  🤔 
Try with french labels for example: "Tous, Propriétaire, Membre, Privé, Public"